src/Avalonia.Controls/Converters/CornerRadiusCorner.cs

```diff
@@ -3,29 +3,33 @@
 namespace Avalonia.Controls.Converters
 {
     /// <summary>
-    /// Defines constants that specify the filter type for a <see cref="CornerRadiusFilterConverter"/> instance.
+    /// Defines constants that specify the corner of a <see cref="CornerRadius"/>.
     /// </summary>
     [Flags]
-    public enum CornerRadiusFilterKinds
+    public enum CornerRadiusCorner
```

Comment:
This is a breaking name change of the enum. It was improperly named in the past in my opinion (never should have been plural) and it was also overly specific. Now it can be shared with the new CornerRadiusToDoubleConverter.

Reply:
> never should have been plural

https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN

- Use a singular name for most Enum types, but use a plural name for Enum types that are bit fields.
- Always add the FlagsAttribute to a bit field Enum type.

Reply:
But I agree about "overly specific"

Reply:
Well, I never knew that! Thanks for pointing it out. I guess it makes sense since more bit than one bit field can be set at once.

Would you like me to change it back? Honestly the name isn't great right now with "Corner" twice and I'm not sure changing it to "CornerRadiusCorners" is very good either. Might be better to just call it "Corners".

Reply:
Just 'Corners' seems to work

Reply:
Changed to 'Corners'